src/components/Customizer/Customizer.tsx

```diff
@@ -1,26 +1,19 @@
 /*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
 import '../../yamui';
 import * as React from 'react';
-import { BaseComponentProps } from '../../util/BaseComponent/props';
-import { Customizer as FabricCustomizer } from 'office-ui-fabric-react/lib/Utilities';
+import {
+  Customizer as FabricCustomizer,
+  ICustomizerProps as CustomizerProps,
+} from 'office-ui-fabric-react/lib/Utilities';
 import { createTheme, ITheme } from 'office-ui-fabric-react';
 
-export { createTheme, ITheme };
-
-export interface CustomizerProps extends BaseComponentProps {
-  /**
-   * Theme object.
-   */
-  theme: ITheme;
-}
+export { createTheme, ITheme, CustomizerProps };
 
 /**
  * Used to inject a custom theme object into customizable components, overriding the global theme.
  */
 export default class Customizer extends React.Component<CustomizerProps> {
   public render() {
-    const { theme } = this.props;
-
-    return <FabricCustomizer settings={{ theme }}>{this.props.children}</FabricCustomizer>;
+    return <FabricCustomizer {...this.props} />;
   }
 }
```

Comment:
Why not `export default FabricCustomizer` directly?

Reply:
@unindented I tried, but that removed it from the style guide.  

There are some related docs here, but it wasn't clear to me how else to force it back in.

Reply:
https://react-styleguidist.js.org/docs/thirdparties.html